infrastructure-playbooks/rolling_update.yml

```diff
@@ -326,6 +326,21 @@
         - containerized_deployment
 
 
+- name: set osd flags
+  hosts: "{{ mon_group_name | default('mons') }}[0]"
+  become: True
+  tasks:
+    - import_role:
+        name: ceph-defaults
+    - import_role:
+        name: ceph-facts
+
+    - name: set osd flags
+      command: "{{ docker_exec_cmd | default('') }} ceph --cluster {{ cluster }} osd set {{ item }}"
+      with_items:
+        - noout
+        - nodeep-scrub
```

Comment:
looks like import_role isn't happy with ansible 2.6

Change this by 'roles: + - post_tasks' instead ?